content/en/continuous_integration/pipelines/azure.md

```diff
@@ -32,17 +32,61 @@ further_reading:
 
 The Datadog integration for [Azure Pipelines][1] works by using [service hooks][2] to send data to Datadog.
```

Comment:
Is this still accurate?

> works by using [service hooks][2] ...

Reply:
yes the extension is still service hooks